INSTALL.rst

```diff
@@ -4,9 +4,16 @@ Installation
 
 This installation is meant for developers of Open Forms. If you are looking to
 install Open Forms to try it out, or to run it in production, please consult
-the documentation.
+the `documentation <https://open-forms.readthedocs.io/en/stable/installation/>`_ for
+alternative installation methods.
 
-The project is developed in `Python`_ using the `Django framework`_.
+.. note:: The `Developer documentation`_ on Read the Docs provides working internal
+   links, in case you are reading this on Github or elsewhere.
+
+The (backend) project is developed in `Python`_ using the `Django framework`_.
+
+For the SDK (frontend), see the `Storybook <https://open-formulieren.github.io/open-forms-sdk/>`_
```

Comment:
Can we elaborate a bit more whether this applies to developing custom application using the sdk?

Reply:
It's not there yet, but we should strive towards that, yes. Plain markdown documentation pages in Storybook should make that possible.